tests/integration/test_debug.py

```diff
@@ -264,6 +267,44 @@ def test_tracer_info_level_log(self):
         assert tracer.log.log.mock_calls == []
 
 
+def test_runtime_metrics_enabled_via_manual_start():
```

Comment:
I wonder if this should be a subprocess test to handle any cleanup from enabling runtime metrics 🤔

Reply:
I'll make this change